src/core/lang.c

```diff
@@ -1,6 +1,7 @@
-#include "building/type.h"
 #include "core/lang.h"
 
+#include "building/building.h"
+#include "building/type.h"
```

Comment:
No need to include `building/type.h`. It's already included on `building/building.h`.

Reply:
Fixed.